provider/kubernetes/kubernetes.go

```diff
@@ -323,10 +333,46 @@ func (p *Provider) loadIngresses(k8sClient Client) (*types.Configuration, error)
 				}
 			}
 		}
+
+		err = p.updateIngressStatus(i, k8sClient)
+		if err != nil {
+			log.Printf("cannot update Ingress %s/%s due to error: %s", i.Namespace, i.Name, err)
```

Comment:
should be errorf

Reply:
Not here, because we don't want to stop to browse all other ingress

Reply:
But why would `log.Errorf` stop processing? If it's an error we should mark it that way in the logs, shouldn't we?

Reply:
You mix `%s` and `%v` for errors. I don't really mind as there is no logical difference in the output, but consistence is a thing and `%s` should be faster as it doesn't have to do type assertions and therefore would stick with it.

Reply:
Logs should start with upper case.

Reply:
oh yes sorry I wa thinking about `fmt.Errorf`